website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -35,7 +35,7 @@ compatible Envoy versions.
 
 | Consul Version      | Compatible Envoy Versions                              |
 | ------------------- | ------------------------------------------------------ |
-| 1.11.x              | 1.20.0, 1.19.1, 1.18.4, 1.17.4                         |
+| 1.11.x              | 1.20.1, 1.19.1, 1.18.4, 1.17.4                         |
```

Comment:
Should we still claim compatibility with Envoy 1.20.0 for Consul 1.11.x or did we drop 1.20.0 support between 1.10.x and 1.11.x?

Reply:
We usually only point to the latest Envoy release since older versions contain CVEs and have vulnerabilities